src/main/resources/static/js/subscription.js

```diff
@@ -896,6 +902,10 @@ jQuery(document).ready(function () {
             var emails = notificationMeta.split(",");
             for(var email of emails) {
                 email = email.trim();
+                // When this regExp need to be changed then remember to change the one in the
+                // back-end (validEmailRegExpression in SubscriptionValidator.java), which do the same
+                // email validation check. The two
+                // regEx always need to be the same for ensuring the same check.
```

Comment:
Manual test some mail addresses that should succeed and some mail addresses that should fail before merge.